src/modules/windowwalker/app/Window Walker/ViewModels/WindowWalkerViewModel.cs

```diff
@@ -226,7 +226,8 @@ private void WindowHide()
         {
             LivePreview.DeactivateLivePreview();
             WindowVisibility = false;
-            ApplicationUpdates.InstallUpdateSyncWithInfo();

```

Comment:
Found my useless method call have you?

Reply:
I saw it and wanted to be 100% sure you were not doing anything :)